03.macos/05.solving-problems/04.big-sur-issues/docs.en.md

```diff
@@ -100,7 +100,18 @@ Despite AdGuard is displayed as a VPN in system settings, it shouldn't cause any
 <img src="https://cdn.adguard.com/public/Adguard/kb/BigSur/problems/legacy-api.png" style="max-width: 650px;">
 
 
+<a id="little-snitch"></a>
+
+### Compatibility with Little Snitch 5
+
+At this moment, Network Extension mode in AdGuard isn't compatible with [Little Snitch 5](https://obdev.at/products/littlesnitch/index.html). When both are running, there's a chance to encounter issues with various apps' behavior, even if they aren't filtered by AdGuard. This problem is directly linked to Big Sur, and we've already informed Apple about it.
```

Comment:
We might want to mention that the issue is caused by a bug in MacOS which we already reported to Apple and there's a chance that it will be resolved in the next versions.